Source/WebCore/css/calc/CSSCalcValue.cpp

```diff
@@ -186,6 +188,9 @@ inline double CSSCalcValue::clampToPermittedRange(double value) const
     if (m_tree.category == Calculation::Category::Angle && std::isinf(value))
         return 0;
 
+    if (m_tree.category == Calculation::Category::Integer)
+        value = std::floor(value + 0.5);
```

Comment:
Isn't there a more efficient way to do this operation?

Reply:
Not sure. This is just moving from elsewhere. 

Going to take a look at the assembly and see what the compiler outputs for it verse other rounding functions.